jovian/utils/commit.py

```diff
@@ -129,10 +129,19 @@ def commit(message=None,
         if project is None:
             log("Please provide the project argument e.g. jovian.commit(project='my-project')", error=True)
             return
+        if environment == "conda":
+            log("Colab uses a pip environment, cannot capture conda environment. Will Capture pip environment instead.")
```

Comment:
Could simplify this:
```suggestion
            log("Conda environment is not supported on Colab. Capturing pip environment instead.")
```